vendor/assets/javascripts/data-confirm-modal.js

```diff
@@ -151,9 +152,9 @@
     var id = 'confirm-modal-' + String(Math.random()).slice(2, -1);
     var fade = settings.fade ? 'fade' : '';
     var modalClass = options.modalClass ? options.modalClass : settings.modalClass;
+    var modalClose = settings.modalClose;
```

Comment:
Hi, thank you. This should read:

```js
var modalClose = options.modalClose ? options.modalClose : settings.modalClose;
```

So to allow an user to specify a `modalClose` option when instantiating the modal.